core/message/msgreject.go

```diff
@@ -28,6 +28,7 @@ const (
 	RejectDust            RejectCode = 0x41
 	RejectInsufficientFee RejectCode = 0x42
 	RejectCheckpoint      RejectCode = 0x43
+	RejectBan             RejectCode = 0x50
```

Comment:
```
return fmt.Sprintf("Unknown RejectCode (%d)", uint8(code))
```
better to use the hex code instead of using `%d`,  better to keep in consist with code for msg display.

Reply:
https://github.com/Qitmeer/qitmeer/pull/255/commits/b8470458f82c4b88dfa42aa2e84fa3b7026b8f20